### PR TITLE
Create megan_kraus.md

### PR DESCRIPTION
**Pull request by megankraus**
I have checked that:

- [x] all pages' content, images, and styles still appear correctly
- [x] all links still work
- [x] redirects (e.g `deploy-preview-XX--tislab-website.netlify.app/melzoom`) are still working

### CODE REVIEW

**Comment by megankraus** [comment]
Need to add images/team/megan_kraus.jpg to image section

File: _members/megan_kraus.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+name: Megan Kraus
+image:
```
Comment (monicacecilia) [inline _members/megan_kraus.md]: Edit to: 
`image: images/team/megan_kraus.jpg`

**Review by monicacecilia — COMMENTED** [review]
Just one little thing to add to your file, and it will be ready! 😊

File: _members/megan_kraus.md (hunk 1 above)
Comment (megankraus) [reply _members/megan_kraus.md]: Complete

**Review by monicacecilia — APPROVED** [review]

**Pull request merged**